### ISSUE TITLE
Tryb hardcore.

### ISSUE THREAD

**Issue by Wruczek** [Wruczek/FunnyGuilds-Reloaded#7]
Co myślicie o wbudowanym trybie hardcore?
Gracz po śmierci otrzymywałby bana za określony czas. Oczywiście możliwość wyłączenia w configu.

https://github.com/grzegorz2047/OpenGuild2047/wiki/Modu%C5%82-Hardcore

**Comment by TheMolkaPL** [comment]
Myślę, że powinniśmy najpierw skupić się nad #5 i błędami/ulepszeniami tam wymienionymi.
Aczkolwiek nie twierdzę, że jest to fajna funkcja i mogłaby się znaleźć w tym pluginie.

**Comment by Wruczek** [comment]
Tak, to taki pomysł na potem, do sekcji "Po napisaniu bazy danych".
